internals/cli/ui/ask_test.go

```diff
@@ -244,15 +244,36 @@ func TestChoose(t *testing.T) {
 			in: []string{"\n", "2\n"},
 
 			expected: "bar",
-			out:      "foo? Press [ENTER] for options.\nfoo?\n1) foo\n2) bar\n3) baz\n",
+			out: "foo? Press [ENTER] for options.\n" +
+				"1) foo\n" +
+				"2) bar\n" +
+				"3) baz\n" +
+				"Give number of choice or enter other option: \n",
```

Comment:
Two things that aren't immediately clear about the current wording:

- `Give number` is ambiguous, `type a number` is more direct.
- `enter other option` is ambiguous: the options are numbered so I can enter a different number than on the list or I can enter a different answer to the question (in our case a `region`)? `or enter a different %s` can be clearer. 

Instead of saying `Give number of choice or enter other option:` could we say `Type a number of choice or enter a different [region|foo|option]:`

Reply:
> the options are numbered so I can enter a different number than on the list or I can enter a different answer to the question (in our case a region)?

Haha, it's intended as being a number from the list or a value (can be from the list).

e.g.:
1) foo
2) bar

Inputs can be: `1`, `2`, `foo` and `bar`, but also `baz`.

In the case of regions, only values from the list are valid, as we show all regions.

So I would omit `different` from the suggested phrase, to put let emphasis on that. How do you like that?

`Type a number of choice or enter a [region|foo|option]:`

Reply:
Nice, maybe `Type a number of choice or enter a value` is clear enough, but I like it also with `region` in there.